djangocms_admin_style/sass/components/_content.scss

```diff
@@ -268,6 +268,12 @@ ul.messagelist + #content {
             border: 0 !important
         }
     }
+    label {
+        vertical-align: middle !important;
+    }
+    input[type="checkbox"] {
+        vertical-align: middle !important;
+    }
```

Comment:
if there are no additional css, you can combine `label` & `input[type="checkbox"]`